projects/smartseq2/CreateSs2AdapterMetadata.wdl

```diff
@@ -199,6 +207,7 @@ workflow CreateSs2AdapterMetadata {
   Array[File] reference_metadata_objects = CreateReferenceMetadata.reference_metadata_outputs
   Array[File] reference_file_descriptor_objects = CreateReferenceMetadata.reference_file_descriptor_outputs
   Array[File] data_objects = flatten([reference_fasta_array, [output_loom], output_bams, output_bais])
+    Array[File] is_update_file = CreateStagingAreaFile.is_update_file
```

Comment:
```suggestion
    File is_update_file = CreateStagingAreaFile.is_update_file
```

Reply:
additionally spacing looks like it might be off for this line